public/netsuke/assets/vendor/minisearch.js

```
@@ -0,0 +1,2018 @@
+/*!
+ * MiniSearch v7.2.0
+ * @see https://github.com/lucaong/minisearch
+ * @license MIT
+ */
+(function (global, factory) {
+    typeof exports === 'object' && typeof module !== 'undefined' ? module.exports = factory() :
```

Biome (2.4.9)

[error] The assignment should not be in an expression.

(lint/suspicious/noAssignInExpressions)